### PR TITLE
Zendesk airbyte entrypoint

### PR DESCRIPTION
**Pull request by marcosmarxm**
## What
*Describe what the change is solving*
*It helps to add screenshots if it affects the frontend.*

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## Pre-merge Checklist
Expand the checklist which is relevant for this PR. 

<details><summary> <strong> Connector checklist </strong> </summary>
<p>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Secrets are annotated with `airbyte_secret` in the connector's spec
- [ ] Credentials added to Github CI if needed and not already present. [instructions for injecting secrets into CI](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#using-credentials-in-ci). 
- [ ] Unit & integration tests added as appropriate (and are passing)
    * Community members: please provide proof of this succeeding locally e.g: screenshot or copy-paste acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] `/test connector=connectors/<name>` command as documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
    * Community members can skip this, Airbyters will run this for you. 
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] `README.md`
    - [ ] `docs/SUMMARY.md` if it's a new connector
    - [ ] Created or updated reference docs in `docs/integrations/<source or destination>/<name>`.
    - [ ] Changelog in the appropriate page in `docs/integrations/...`. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md` contains a reference to the new connector
    - [ ] Build status added to [build page](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/builds.md)
- [ ] Build is successful
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] No major blockers
- [ ] PR merged into master branch
- [ ] Follow up tickets have been created
- [ ] Associated tickets have been closed & stakeholders notified
</p>
</details>

<details><summary> <strong> Connector Generator checklist </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-zendesk-support-singer/Dockerfile [hunk 1]
```diff
@@ -5,11 +5,12 @@ RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
 ENV CODE_PATH="source_zendesk_support_singer"
 ENV AIRBYTE_IMPL_MODULE="source_zendesk_support_singer"
 ENV AIRBYTE_IMPL_PATH="SourceZendeskSupportSinger"
+ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/source.py"
```
Comment (jrhizor) [inline airbyte-integrations/connectors/source-zendesk-support-singer/Dockerfile:8]:
```suggestion
ENV AIRBYTE_ENTRYPOINT "/airbyte/base.sh"
```

Comment (jrhizor) [reply airbyte-integrations/connectors/source-zendesk-support-singer/Dockerfile:8]: `docker inspect airbyte/source-zendesk-support-singer:0.2.2` shows `/airbyte/base.sh` as the existing entrypoint.

**Review by jrhizor — CHANGES_REQUESTED** [review]

**Review by jrhizor — APPROVED** [review]

**Comment by jrhizor** [comment]
looks like this probably has to be pushed with skipping tests with `/publish connector=connectors/source-zendesk-support-singer run-tests=false`

**Comment by marcosmarxm** [comment]
/publish connector=connectors/source-zendesk-support-singer run-tests=false

**Comment by marcosmarxm** [comment]
> Failed to apply plugin 'airbyte-python'.
   > A problem occurred configuring project ':airbyte-integrations:connectors:source-zoom-singer'.
      > Could not open cp_proj generic class cache for build file '/home/runner/work/airbyte/airbyte/airbyte-integrations/connectors/source-zoom-singer/build.gradle' (/home/runner/.gradle/caches/6.7.1/scripts/4ffd4lk4g399iqzx3xoc30xgr).
         > java.lang.StackOverflowError (no error message)

I dont understand the CI error :( @sherifnada any idea why CI is failing to publish zendesk support and is calling zoom?

**Comment by sherifnada** [comment]
not sure about zoom but previous failed runs of this workflow have been because the schema of the configured catalog used in testing is out of date, requires destination_sync_mode

**Pull request merged**